packages/http/src/mocker/__tests__/HttpMocker.spec.ts

```diff
@@ -127,7 +127,28 @@ describe('mocker', () => {
       });
     });
 
-    describe('with invalid negotiator response', () => {
+    describe('with a negotiator response containing validation results of Warning severity', () => {
+      it('returns static example', () => {
+        jest.spyOn(helpers, 'negotiateOptionsForValidRequest').mockReturnValue(
+          right({
+            code: '202',
+            mediaType: 'test',
+            bodyExample: mockResource.responses![0].contents![0].examples![0],
+            headers: [],
+          }),
+        );
+
+        const mockResult = mock({
+          config: { dynamic: false },
+          resource: mockResource,
+          input: Object.assign({}, mockInput, { validations: [{ severity: DiagnosticSeverity.Warning }] }),
+        })(logger);
+
+        assertRight(mockResult, result => expect(result).toMatchSnapshot());
```

Comment:
Since the result is mocked, instead of adding a snapshot — I'd just make sure `negotiateOptionsForValidRequest ` was called and `negotiateOptionsForInvalidRequest ` was NOT called.